src/neptune/versionHelper.d

```diff
@@ -0,0 +1,334 @@
+/*******************************************************************************
+
+    Helper to deal with version strings
+
+    Copyright:
+        Copyright (c) 2009-2016 Sociomantic Labs GmbH. All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE.txt for details.
+
+*******************************************************************************/
+
+module neptune.versionHelper;
```

Comment:
Well this isn't really a separate issue - I wouldn't be particurarly happy if this PR was merged with two separate version helpers. It was OK for rapid prototype but they should be unified before the merge.